specification/containerservice/Fleet.Management/update/run.tsp

```diff
@@ -135,6 +136,9 @@ enum ManagedClusterUpgradeType {
 
   @doc("NodeImageOnly upgrades only the node images of the target ManagedClusters.")
   NodeImageOnly,
+
+  @doc("ControlPlaneOnly upgrades only the control plane of the target ManagedClusters.")
```

Comment:
If user selects this option, what of the CP do we upgrade, k8s version or node image version?